root/static/scripts/edit/components/forms.js

```diff
@@ -59,3 +59,34 @@ MB.initializeArtistCredit = function (form, initialArtistCredit) {
     );
   });
 };
+
+/*
+ * Registers a beforeunload event listener on the window that prompts
+ * the user if any of the page's form inputs have been changed.
+ */
+MB.installFormUnloadWarning = function () {
+  let modified = false;
+
+  const form = document.querySelector('#page form');
```

Comment:
Would it be better to have callers pass the form element into this function? Otherwise, should I add a null check here and e.g. log an error and return? I'm not sure whether the `#page form` selector is expected to continue to work for all forms, although it's used in various other files.